content/en/api/downtimes/downtimes_schedule.md

```diff
@@ -26,6 +26,9 @@ external_redirect: /api/#schedule-monitor-downtime
 * **`recurrence`** [*optional*, *default*=**None**]:
     An object defining the recurrence of the downtime with a variety of parameters:
     *   **`type`** the type of recurrence. Choose from: `days`, `weeks`, `months`, `years`.
+      *   **`rrule`** standard for defining recurring events. For example to have a recurring event each month, select a type of `rrule` and a `FREQ` of `MONTHLY`. Any RRULE according to RFC is supported from this API call.
```

Comment:
* Is this indented too far? `rrule` isn't a sub-option of `type`.
* Might be good to include a full valid rrule, e.g. `FREQ=MONTHLY;BYMONTHDAY=1`
* Can we loosen the language about supporting the full RFC, maybe `Most common RRULE options from the [iCalendar Spec](https://tools.ietf.org/html/rfc2445#section-4.8.5.4) are supported.`